core/src/main/java/me/leoko/advancedban/MethodInterface.java

```diff
@@ -250,6 +250,13 @@
      */
     boolean callCMD(Object player, String cmd);
 
+    /**
+     * Check if the given Sender object is a consoleSender
+     *
+     * @param sender the sender
+     * @return the boolean
```

Comment:
I'm not sure whether Leoko is picky about this, so I'll let it be.

Reply:
Tried to match the other notes from the other methods in this class, I'd be free to make any changes if Leoko wants to